### PR TITLE
Alexei patch

### PR DESCRIPTION
**Pull request by alexeiva**
- Adding missing Vietnamese characters
- Adding .sinf glyphs to Pro set
- Adding foursuperios
- Adding tutorials section with tips to speed up the charset expansion process

### CODE REVIEW

**Comment by davelab6** [comment]
@kalapi please review :)

**Comment by alexeiva** [comment]
This fixes #316.
@davelab6 I have fixed the merge conflict in this branch with master. 
How is it possible to merge this branch, and squash all my commits in one?

**Comment by davelab6** [comment]
The merge button, when clicked, offers a dropdown for either "squash and merge" or "confirm merge", and the former then offers a way to edit the commit message:

screen shot 2016-07-11 at 23 19 17

**Comment by alexeiva** [comment]
Thanks for clarifying, @davelab6 

Here is what I see: no triangle. Will the triangle be as a second step?

Anyway, I am not pressing the big green button _this_ time.
screen shot 2016-07-12 at 00 15 29

**Comment by kalapi** [comment]
@alexeiva Please refer to #316 and make the necessary changes if you agree. We can discuss this in more detail if you like.

**Comment by davelab6** [comment]
Yes is appearing  in a second step

File: tools/encodings/GF 2016 Glyph Sets/FilterLists/Google Latin Expert.txt [hunk 1]
```diff
@@ -372,6 +372,15 @@ breveinvertedcomb
...
 dotbelowcomb
+strokeshortcomb
+brevecomb_acutecomb
+brevecomb_gravecomb
+brevecomb_hookabovecomb
+brevecomb_tildecomb
+circumflexcomb_acutecomb
+circumflexcomb_gravecomb
+circumflexcomb_hookabovecomb
+circumflexcomb_tildecomb
```
Comment (kalapi) [inline tools/encodings/GF 2016 Glyph Sets/FilterLists/Google Latin Expert.txt]: I'm not sure if all these are needed. These can be accessed through mark2mark attachment. Unless i'm missing something?

File: tools/encodings/GF 2016 Glyph Sets/FilterLists/Google Latin Plus.txt [hunk 2]
```diff
@@ -372,6 +372,15 @@ breveinvertedcomb
...
 dotbelowcomb
+strokeshortcomb
+brevecomb_acutecomb
+brevecomb_gravecomb
+brevecomb_hookabovecomb
+brevecomb_tildecomb
+circumflexcomb_acutecomb
+circumflexcomb_gravecomb
+circumflexcomb_hookabovecomb
+circumflexcomb_tildecomb
```
Comment (kalapi) [inline tools/encodings/GF 2016 Glyph Sets/FilterLists/Google Latin Plus.txt]: Similarly, I'm not sure if all these are needed either.

File: tools/encodings/GF 2016 Glyph Sets/FilterLists/Google Latin Pro.txt [hunk 3]
```diff
@@ -372,6 +372,15 @@ breveinvertedcomb
...
 dotbelowcomb
+strokeshortcomb
+brevecomb_acutecomb
+brevecomb_gravecomb
+brevecomb_hookabovecomb
+brevecomb_tildecomb
+circumflexcomb_acutecomb
+circumflexcomb_gravecomb
+circumflexcomb_hookabovecomb
+circumflexcomb_tildecomb
```
Comment (kalapi) [inline tools/encodings/GF 2016 Glyph Sets/FilterLists/Google Latin Pro.txt]: Similarly, I'm not sure if all these are needed either.

Comment (alexeiva) [inline tools/encodings/GF 2016 Glyph Sets/FilterLists/Google Latin Pro.txt]: I found 'circumflexcomb_acutecomb' *very* useful for design-cases where custom Vietnamese accents are requiered. All these characters were borrowed from the Vietnamese set in Glyph App.

Technically they aren't necessary for font generation.
However, I think specifying them here will encourage designers to pay more attention
to Vietnamese.  Or at least list them in the recommended section, and hope that people _will_ read it.

@crystaltype What do you think?

screen shot 2016-07-20 at 12 21 50

**Comment by alexeiva** [comment]
@kalapi, I have reconsidered the `.subs` problem. What do you think of this proposal?

screen shot 2016-07-20 at 12 30 26

**Comment by davelab6** [comment]
(Why is your proposal an image? :)

**Comment by alexeiva** [comment]
It's accessible here: https://github.com/google/fonts/issues/316

**Comment by alexeiva** [comment]
New suggestion, put these glyphs in a separate
`GT-Latin-Plus-better-vietnamese.txt` list

```
strokeshortcomb
brevecomb_acutecomb
...
circumflexcomb_tildecomb
```

**Comment by alexeiva** [comment]
I have a new proposal:

Remove these ligatures from GF Latin Plus to the recommended list:
```
f_f_ij
f_ij
```	

Their use is only justified in case of a truly ligated `ij` in the font. Non of my `ij` diagraphs are ligated in the three fonts I worked on recently, adding them was redundant.


In other words `f_f_i` + `j` produces same results as the said `f_f_ij` ligature, rendering them unnecessary in the default use-cases.

_Top: `f_f_i` + `j` Bottom: `f_f_ij` ligature. Results are identical._
screen shot 2016-07-28 at 18 43 53

Only fonts with custom ligated `ij`'s, such as perhaps Pacifico may require these ligatures.

**Comment by davelab6** [comment]
On 28 July 2016 at 18:46, Alexei Vanyashin <notifications@github.com> wrote:

> Only fonts with custom ligated ij's, such as perhaps Pacifico may require
> these ligatures.
>
>
Makes sense :) Please file issues with the glyphs sets definition on
www.github.com/google/fonts/issues in the 'Glyph Set Definitions' milestone
:)

**Comment by davelab6** [comment]
@kalapi @alexeiva what is needed to merge this PR?

**Comment by alexeiva** [comment]
@davelab6 I am happy for this to merge now. This update is becoming critical (Jacque's problems with IJ acute, etc)

**Pull request merged**